Thesis.tex

```diff
@@ -58,26 +58,22 @@ \chapter{Introduction}
 
-economic rights holder can authorize/prohibit
-reproduction, broadcast, public performance, recording, translation, adaptation
+According to the berne convention copyright is obtained automatically with the act of creation and usually covers a very long time span. There is no registration needed before one can claim and enforce copyright over their own works. Often voluntary registration is available nonetheless to help solve disputes over ownership. Such registrations can help facilitate financial transactions related to a copyright and assignment or transfer of rights to other parties. There exist a number of organizations offering collective management of copyrights, exercising those rights and related transactions on behalf of the rights owner.
 
-according to the berne convention copyright is obtained automatically, without registration
-usually covers a very long timespan
-voluntary registration is often available, can help solve disputes over ownership, facilitate financial transactions, assignment/transfer of rights
+\subsection{Copyright on the internet}
+Many users of the internet have a number of misconceptions about copyright and how it applies to their usage of internet services. The most important misconceptions are outlined in an article by Vorys at al. \cite{Vorys2013}.
 
-collective management of copyright, exercise of copyrights and similar rights by organizations on behalf of rights owner
+Probably the most common misunderstanding is that everything found on the internet can be freely downloaded and used as the basis of own works. However this is nit true in most cases. As mentioned in chapter \ref{sec:CopyDef} copyright is applied automatically to all eligible works and unlicensed reuse is prohibited unless explicitly authorized by the rights owner. The source of the works and whether you payed for i or not -- for instance an iTunes purchase versus a YouTube download -- is not relevant.
 
-\subsection{Copyright on the internet}
-common misconception: everything on the internet can be downloaded and used freely
+Another very common misconception often seen on YouTube is that it suffices to either give credit to the original creator, add a copyright notice or claim so called fair use. Adding a copyright notice or crediting the original author has no legal meaning by itself. roper authorization to use the work is still required.
+
+The aforementioned fair use is also a source for confusion among internet users. Fair use is a doctrine that under certain circumstances allows the use of copyrighted material without acquiring permission from the rights holder. It is intended as a defence against law suits and can not be regarded as a safe harbour. If fair use is applicable has to be decided by a court on a case by case basis (\cite{WikiFair} and \cite{Vorys2013}). As such claiming fair use for instance in the description text of a YouTube video is meaningless.
```

Comment:
Stimmt so nicht ganz. Es ist zwar eine defense (mit s geschrieben). Zwar muss das Gericht Fall für Fall beurteilen, 

In Lenz v. Universal Music Group wird ersichtlich, dass Fair Use nicht nur ein "erlaubter Verstoss" sondern dass es sich um erlaubte Aktionen handelt:

Here, the Court concludes that the plain meaning of "authorized by law" is unambiguous. An activity or behavior "authorized by law" is one permitted by law or not contrary to law. Though Congress did not expressly mention the fair use doctrine in the DMCA, the Copyright Act provides explicitly that "the fair use of a copyrighted work ... is not an infringement of copyright."   https://h2o.law.harvard.edu/cases/5589

Weiterhin ist es so, dass im Falle eines Gerichtsprozesses das Gericht entscheiden muss, auch hier gibt es aber Vorgaben, so, aus dem Selben Urteil

The Copyright Act unequivocally establishes the four factors used to determine fair use:

    In determining whether the use made of a work in any particular case is a fair use the factors to be considered shall include—

    (1) the purpose and character of the use, including whether such use is of a commercial nature or is for nonprofit educational purposes;

    (2) the nature of the copyrighted work;

    (3) the amount and substantiality of the portion used in relation to the copyrighted work as a whole; and

    (4) the effect of the use upon the potential market for or value of the copyrighted work.

The fact that a work is unpublished shall not itself bar a finding of fair use if such finding is made upon consideration of all the above factors.

Es stimmt, dass im legalen Sinne eine description mit Hinweis auf Fair Use nicht genügt. Allerdings wäre es die Pflicht der Rechteinhaber vor einer Abmahnung, DMCA oder ähnlichem die Anwendbarkeit von Fair Use prüfen müssten (steht irgendwo im gleichen lenz vs UMG Urteil